x/utilities/bimap_with_default.go

```diff
@@ -0,0 +1,19 @@
+package utilities
```

Comment:
Needs a more descriptive name, ideally in another package (outside of x/). I would suggest keeping this with the symbol translator package / file.

Reply:
Moved the package.